### ISSUE TITLE
Module parse failed

### ISSUE THREAD

**Issue by andersbc** [angular-class/angular2-webpack-starter#33]
I get this error in the terminal after installing and  running ~~`npm server start`~~ `npm run server`
I'm on ubuntu 15.04, node: v0.12.7, npm: ~~v0.12.7~~ 2.11.3
An earlier version of the repo didn't produce this error, som I guess it has something to do with recent changes. I'm too new to angular to find the root of the problem my self. 

Error:

```
ERROR in ./src/app/bootstrap.ts
Module parse failed: /home/anders/Dropbox/dev/itembox/code/webpacktest/src/app/bootstrap.ts Line 4: Unexpected reserved word
You may need an appropriate loader to handle this file type.
| 
| // Angular 2
| import {bootstrap} from 'angular2/angular2';
| 
| 
 @ multi app

ERROR in ./src/app-simple/bootstrap.ts
Module parse failed: /home/anders/Dropbox/dev/itembox/code/webpacktest/src/app-simple/bootstrap.ts Line 4: Unexpected reserved word
You may need an appropriate loader to handle this file type.
| 
| // Angular 2
| import {bootstrap} from 'angular2/angular2';
| /*
|  * Angular Modules
 @ multi app-simple
```

**Comment by gdi2290** [comment]
can you delete your `node_modules/` and run `npm install` I changed the version of typescript

**Comment by gdi2290** [comment]
and also ensure this file is correct https://github.com/angular-class/angular2-webpack-starter/blob/master/webpack.config.js#L98

**Comment by andersbc** [comment]
It didn't fix it :-(

I did as you suggested (also ran `npm cache clean` before npm install). webpack.config.js is identical to the current one.

Here are some snippets of what npm packages are installed in the two (I used `npm list`):

In my working version of the repo (I think its 2 commits before the current one) I have:
```
...
+-- typescript@1.5.3 (git://github.com/microsoft/TypeScript.git#a24aa6f57d281bafd3535f0f4c3f492364a8b4d6)
+-- typescript-simple-loader@0.3.0
|     +-- loader-utils@0.2.10
|     |     +-- big.js@3.1.3 
|     |     +-- json5@0.4.0
|     +-- xtend@4.0.0  
...
```
In the current version - the one giving me the error, I have:
```
...
+-- typescript@1.5.3 (git://github.com/microsoft/TypeScript.git#a24aa6f57d281bafd3535f0f4c3f492364a8b4d6)
+-- typescript-simple-loader@0.3.2
|     +-- arrify@1.0.0
|     +-- loader-utils@0.2.11
|     |     +-- big.js@3.1.3 
|     |     +-- json5@0.4.0
|     +--  tsconfig@1.0.1 
...
```

**Comment by gdi2290** [comment]
cc @blakeembrey

**Comment by andersbc** [comment]
sorry.. push the wrong button -- reopening the issue

**Comment by blakeembrey** [comment]
@andersbc What happens when you use an older loader version like `npm i typescript-simple-loader@0.3.0`? The only possible commit that (I think) can be causing issues is https://github.com/blakeembrey/typescript-simple-loader/commit/db5a44edb92e7e0b3f80f50c83a1dc315267c3cc

**Comment by andersbc** [comment]
@blakeembrey. thanks, it wasn't that though (it didn't change anything). I think the problem comes from some npm-related fault on my part when I installed the repo. I did a completely fresh install (git clone) and the problem disappeared. **A diff between the working and failing installation pointed to some differences in node locations, so I'm going to close the issue, since it is not related to this repo.**  

For the record:
The diff between my failing install and my new fresh install came up with a difference in a config.gypi file. In the working case the 'prefix' variable had the value of ''/usr/local". In the failing case it was "/home/anders/.nvm/versions/node/v0.12.7" (there were other differences in this file as well). If I recall right I followed the instructions in [this article](https://www.digitalocean.com/community/tutorials/how-to-install-node-js-with-nvm-node-version-manager-on-a-vps) between the failing and working installation. There are a command that makes the currently active node version (I'm using [node version manager](https://github.com/creationix/nvm)) global to all users - my hunch is that is that that command fixed my problem.
